packages/manager/modules/vps/src/dashboard/components/vps-announcement-banner/controller.js

```diff
@@ -0,0 +1,14 @@
+import { TASKS_INFO } from './vps-announcement-banner.constant';
+
+export default class VpsAnnouncementBannerComponentController {
+  /* @ngInject */
+  constructor(coreConfig) {
+    this.coreConfig = coreConfig;
+  }
+
+  getTasksInfoLink() {
+    return (
+      TASKS_INFO[this.coreConfig.getUser().ovhSubsidiary] || TASKS_INFO.DEFAULT
```

Comment:
Why not do this in `$onInit` instead and bind the link to the UI, instead of calling the method from the UI?

Reply:
because it's more clear i think.
do you expect something ?

Reply:
My point is, this method does not have to execute more than once, since it is going to give the same result... Why not do it in `$onInit`